app/views/admin/hearing_days/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="nested-fields">
+  <div class="field">
+    <%= f.hidden_field :hearing_id, value: @hearing.id %>
+    <%= f.label :hearingId, 'Hearing Id' %>
+    <%= f.text_field :hearing_id, disabled: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :sittingDay, 'Sitting Day' %>
+    <%= f.text_field :sittingDay %>
+  </div>
+
+  <div class="field">
+    <%= f.label :listingSequence, 'Listing Sequence'%>
+    <%= f.text_field :listingSequence %>
+  </div>
+
+  <div class="field">
+    <%= f.label :listedDurationMinutes, 'Listed Duration Minutes' %>
+    <%= f.text_field :listedDurationMinutes %>
+  </div>
+
+  <div class="field">
+    <%= f.label :startTime, 'Start time' %>
+    <%= f.text_field :startTime %>
+  </div>
+
+  <div class="field">
+    <%= f.label :endTime, 'End time' %>
+    <%= f.text_field :endTime %>
+  </div>
+
+  <div class="field">
+    <%= f.label :court_centre_id, 'Court Centre' %>
+    <%= f.select :court_centre_id, HmctsCommonPlatform::Reference::CourtCentre.all.collect{ |cc| [cc.oucode_l3_name, cc.id ] } %>
+  </div>
+
```

Comment:
Add field for court_room_id - I don't think we have a list of these anywhere, so just make it a text field and factorybot can just put in a uuid

Reply:
add checkbox for isCancelled
 
 <div class="field">
    <%= f.label :isCancelled, 'Is Cancelled' %>
    <%= f.check_box :isCancelled %>
  </div>